test/Formats/MetadataSyntax.jl

```diff
@@ -75,10 +69,15 @@ Example:
 
 !!section(Lexicon.md/Exported/Methods/1)
 """
-function example14() end
+function example12() end
```

Comment:
As I mentioned in the other comments on that recent commit, changing these definitions to have something in the body, or just using single line ones will get them picked up.